### PR TITLE
Leaf/b1.5 email confirmation

### PR DESCRIPTION
**Pull request by JRSiwiecki**
Issue: [#74](https://github.com/LetsMesh/Site/issues/74)

When a user visits an email confirmation url path, an email is sent to the user, and a unique token is generated and sent as a url to the user, and saved in the settings table. If the saved token and the token in the url matches, then the user's email is confirmed.

### CODE REVIEW

**Comment by JRSiwiecki** [comment]
Likely needs to be updated when issue [#218](https://github.com/LetsMesh/Site/issues/218) is closed, as currently the email confirmation process does not use the database as it should.

File: mesh/confirmation/urls.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from django.urls import path, include
+from . import views as confirmation_views
+
+urlpatterns = [
+    path('<str:user_email>/', confirmation_views.email_confirmation, name="email_confirmation"),
+    path('<str:user_email>/<str:token>/', confirmation_views.confirm_token, name="confirm_token"),
```
Comment (sosropartono) [inline mesh/confirmation/urls.py:6]: Should we have a path specifically for emails? Like /emails/<str...

**Review by sosropartono — APPROVED** [review]
One question, otherwise LGTM

File: mesh/confirmation/urls.py (hunk 1 above)
Comment (JRSiwiecki) [reply mesh/confirmation/urls.py:6]: I figure that the confirmation path in the urls.py in the root mesh folder where it is already confirmation/<str:user_email>/ handles this idea? Unless you mean add the /email/ on top of the confirmation path for clarity.

**Review by JosephKahale — APPROVED** [review]
LGTM

**Pull request closed without merging**